pkg/proxy/winkernel/proxier.go

```diff
@@ -685,10 +688,9 @@ func NewProxier(
 		klog.InfoS("ClusterCIDR not specified, unable to distinguish between internal and external traffic")
 	}
 
-	isIPv6 := netutils.IsIPv6(nodeIP)
-	ipFamily := v1.IPv4Protocol
-	if isIPv6 {
-		ipFamily = v1.IPv6Protocol
+	isIPv6 := false
+	if ipFamily == v1.IPv6Protocol {
+		isIPv6 = true
```

Comment:
This gets stored as `proxier.isIPv6Mode`, which is now redundant... you could replace all the `proxier.isIPv6Mode` checks with `proxier.ipFamily == v1.IPv6Protocol`...

It also would be nice to split "add an ipFamily field to the winkernel proxier" out as a separate commit before the "dual stack health checker" commit